zerver/tests/test_typing.py

```diff
@@ -78,6 +78,43 @@ def test_single_recipient(self) -> None:
         self.assertEqual(event['type'], 'typing')
         self.assertEqual(event['op'], 'start')
 
+    def test_single_recipient_by_user_id(self) -> None:
+        """
+        Sending typing notification to a single recipient (using user IDs)
+        is successful
+        """
+        sender = self.example_user('hamlet')
+        recipient = self.example_user('othello')
```

Comment:
call this `recipient_user`; we generally reserve `recipient` for Recipient objects.

Reply:
Done :)